client/src/Component/ChartControl.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { Button, Card, Col, Input, Pagination, ProgressBar, Row } from 'react-materialize';
+
+class ChartControl extends Component {
```

Comment:
Are you using this anywhere?

Reply:
Ok, I like to put todo comments in files like this to that effect (e.g. // todo: decide if I want to use this later)